Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Paged/WalletEthereumProviderScript.js

```diff
@@ -93,7 +93,9 @@ if (window.isSecureContext) {
         return post('isUnlocked', {})
       }),
     }
-  });
+  };
+  $Object.defineProperty(window, 'ethereum', provider);
+  $Object.defineProperty(window, 'braveEthereum', provider);
```

Comment:
We need to define it now because [this check in core ethereum script](https://github.com/brave/brave-core/blob/eaf5cf705d25fc3ca26426cdb44d6e793d121926/components/brave_wallet/resources/ethereum_provider.js#L7) was updated from `window.ethereum` to `window.braveEthereum` for EIP-6963 support. Since we didn’t have `braveEthereum` before (no extensions on iOS), we lost the `EventsEmitter` from the rest of the core script.